.travis.yml

```diff
@@ -29,6 +29,16 @@ script:
 after_success:
   - coveralls
   - python setup.py sdist bdist_wheel
```

Comment:
Why are you removing the sdist & bdist_wheel

Reply:
I didn't, that's how the yml works